app/services/directions_service.rb

```diff
@@ -0,0 +1,17 @@
+class DirectionsService
+
+  def self.get_directions(start_point, dest_lat, dest_long)
+    response = conn.get("/directions/v2/route?from=#{start_point}&to=#{dest_lat},#{dest_long}")
+    JSON.parse(response.body, symbolize_names: true )
+  end
+
+  private
+  def self.conn
+    Faraday.new(url: "http://www.mapquestapi.com" ) do |f|
+      f.params['key'] = ENV['map_api_key']
+    end
+  end
+end
+
```

Comment:
This all looks great! One thing you could do to align a bit more with rails convention is to use block syntax when adding parameters. You did that for your key in your conn method, but you could also do it for your parameters in teh get request on line 4